src/vim9expr.c

```diff
@@ -538,12 +538,11 @@ compile_load_scriptvar(
 	cctx_T *cctx,
 	char_u *name,	    // variable NUL terminated
 	char_u *start,	    // start of variable
-	char_u **end,	    // end of variable, may be NULL
-	imported_T *import) // found imported item, can be NULL
+	char_u **end)	    // end of variable, may be NULL
```

Comment:
@yegappan 
You should not have modified this function.
The fix here eliminates unnecessary calls to `find_imported()` that can occur within `compile_load_scriptvar()` after calling `find_imported()` inside `compile_load()`.

Reply:
Yes.  I missed this optimization that was part of the previous PR.  Thanks for noticing it. The new PR looks good to me.